apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/CreateBucketOptions.cs

```diff
@@ -43,6 +43,11 @@ public sealed class CreateBucketOptions
         /// </summary>
         public RetryOptions RetryOptions { get; set; }
 
+        /// <summary>
+        /// When set to true, object retention is enabled for this bucket.
+        /// </summary>
+        public bool EnableObjectRetention { get; set; }
```

Comment:
This probably should be, and then we only set it if it has a value. This is to allow for explicitly setting one value or another.
```suggestion
        public bool? EnableObjectRetention { get; set; }
```